geode-dunit/src/main/java/org/apache/geode/management/internal/cli/commands/QueryCommandDUnitTestBase.java

```diff
@@ -256,6 +260,15 @@ private static void setupReplicatedRegion(String regionName) {
     assertThat(dataRegion.getFullPath()).contains(regionName);
   }
 
+  private static void setupReplicatedProxyRegion(String regionName) {
+    InternalCache cache = ClusterStartupRule.getCache();
+    RegionFactory regionFactory = cache.createRegionFactory(RegionShortcut.REPLICATE_PROXY);
+
+    Region proxyRegion = regionFactory.create(regionName);
+    assertThat(proxyRegion).isNotNull();
+    assertThat(proxyRegion.getFullPath()).contains(regionName);
+  }
+
```

Comment:
Small nitpicks: since the region name is always the same (`DATA_REGION_WITH_PROXY_NAME`), can we use the constant instead of the parameter?.
Also, to prevent introducing unnecessary warnings, can we avoid the usage of `rawtypes` in the method?:
```
RegionFactory<Integer, Portfolio> regionFactory = cache.createRegionFactory(RegionShortcut.REPLICATE_PROXY);
Region<Integer, Portfolio> proxyRegion = regionFactory.create(regionName);
```

Reply:
Fixed.